build/rive_build_config.lua

```diff
@@ -559,6 +572,37 @@ end
 
 filter({})
 
+-- Cross-compilation helpers for Linux. When targeting a Linux architecture that does not match
+-- the host, add an explicit target triple so clang uses the correct backend. Optionally honor a
+-- provided --sysroot for both compilation and linking.
+if os.target() == 'linux' then
+    local host_arch = os.outputof('uname -m') or ''
+
+    -- Apply target triple if host arch doesn't match requested arch
+    if _OPTIONS['arch'] == 'x64' and host_arch ~= 'x86_64' then
```

Comment:
Thanks for handling these. Can we add x86, just for completeness?

Reply:
I'll try adding it.